webapp/components/wpt-interop.html

```diff
@@ -308,6 +331,10 @@
       return metadata && metadata.test_runs;
     }
 
+    computeInteropNotFound(isLoading, interop) {
+      return !isLoading && !interop;
```

Comment:
nit: Don't rename `passRateMetadata` to `interop.

Won't this yield `interopNotFound = true` until `this.ready()` executes? Perhaps we should observe `isLoading` and check `this.passRateMetadata` exactly when `prevIsLoading = true && isLoading = false`.

Reply:
Yeah, there was a small jank of 'not found' briefly. Done.